### PR TITLE
Add approval queue feature docs

### PR DESCRIPTION
**Pull request by RichardAkman**
Leftovers:
- [x] Updates images from the docs organization

Notes:
- Should we add any special notes regarding upserts being pending approval in the OSS doc pages?
- Should we mention pending approval queue upserts in the editor page, history rollback section, and delete action in config explorer?

### CODE REVIEW

File: content/Platform/configs-request.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+id: configs-request
+slug: configs-request
+title: Configs Request
+---
+
+The Configs Request tab displays all the [protected sets](../settings#protected-sets) and indicates any pending change requests made by members of the organization.
+
+List image here from docs org.
```
Comment (RichardAkman) [inline content/Platform/configs-request.mdx:9]: TODO

File: content/Platform/configs-request.mdx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Admins of the organization can view the pending requests for each protected set and begin a review process to approve or reject each request. As the reviewer approves change requests, the differences between the current and requested values are updated based on their actions but commits only upon reaching the summary stage.
+
+middle of approval queue page img
```
Comment (RichardAkman) [inline content/Platform/configs-request.mdx:13]: TODO

File: content/Platform/configs-request.mdx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+After reviewing all the requests, the reviewer will displayed a summary view to review all the approved changes in bulk and can commit the changes by clicking `upsert`. At any point after the first request, the reviewer may click `aggregated diff` to reach the same summary view.
+
+summary image
```
Comment (RichardAkman) [inline content/Platform/configs-request.mdx:17]: TODO

File: content/Platform/configs-request.mdx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+id: configs-request
+slug: configs-request
+title: Configs Request
```
Comment (pelegpor) [inline content/Platform/configs-request.mdx:4]:
```suggestion
title: Configs Change Request
```

File: content/Platform/configs-request.mdx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+The Configs Request tab displays all the [protected sets](../settings#protected-sets) and indicates any pending change requests made by members of the organization.
+
+![image](./img/configs-request/configs-request-list.png)
+
+Admins of the organization can view the pending requests for each protected set and begin a review process to approve or reject each request. As the reviewer approves change requests, the differences between the current and requested values are updated based on their actions but commits only upon reaching the summary stage.
```
Comment (pelegpor) [inline content/Platform/configs-request.mdx:11]: summary stage - a.k.a aggregated diff

**Comment by pelegpor** [comment]
Please also add explanations on what will happen if I leave the request page and if I want to return to a previous step.
In addition, you missed the blue dot and when it appears and an explanation about the behavior of the Audit Log in this case.

**Pull request merged**